CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.17.0-dev.0
```

Comment:
this version should be 2.18.0-dev.0, as 2.17.0-dev.0 is actually before 2.17.0 in semantic versioning

Reply:
This might need to be updated in our version bumping script.
if we wanted to make a dev version of `2.17.0` what would you expect the dev version to be?

Reply:
Discussed offline! By default we will have the `dev` version be preparatory for next `minor` release